ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/ShowProductUsage.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement, useState } from 'react';
+import { Button, ButtonVariant, Modal, ModalBoxBody, ModalVariant } from '@patternfly/react-core';
+import UsageStatisticsForm from './UsageStatisticsForm';
+
+function ShowProductUsage(): ReactElement {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+
+    function onCloseModalHandler() {
+        setIsModalOpen(false);
+    }
+
+    return (
+        <>
+            <Button
+                key="open-select-modal"
+                data-testid="product-usage-modal-open-button"
+                variant={ButtonVariant.secondary}
+                onClick={() => {
+                    setIsModalOpen(true);
+                }}
+            >
+                Show product usage
+            </Button>
+            <Modal
+                title="Product usage"
+                description="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin blandit augue est. Duis fermentum lacinia consequat. Donec id felis elit. Donec nec nunc felis. Morbi quis enim scelerisque, ullamcorper velit."
+                isOpen={isModalOpen}
+                variant={ModalVariant.large}
+                onClose={onCloseModalHandler}
+                aria-label="Product usage"
+                hasNoBodyWrapper
```

Comment:
First one is with `hasNoBodyWrapper`, second is without. Doc says:
```
Flag indicating if modal content should be placed in a modal box body wrapper.
```

I personally prefer first, otherwise we get a lot of empty space around the elements.

<img width="1785" alt="Screenshot 2023-08-21 at 18 50 17" src="https://github.com/stackrox/stackrox/assets/78353299/4715ae3e-92c8-47ba-b794-17b95a5cce28">
<img width="1776" alt="Screenshot 2023-08-21 at 18 49 53" src="https://github.com/stackrox/stackrox/assets/78353299/6e642b4e-ee14-4468-a60c-66d143ad379d">